chsdi/static/doc/source/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'GeoAdmin API'
-copyright = u'2017, COGIS'
+copyright = u'2013-2019, COGIS'
```

Comment:
what about something like `u'2013-' + datetime.datetime.now().year`? So we don't have to change it ever again

Reply:
Yes, nice idea.